estimagic/optimization/optimize.py

```diff
@@ -492,7 +492,11 @@ def _internal_minimize(
             f"{algorithm} is not a valid choice. Did you mean one of {proposals}?"
         )
 
-    bounds = tuple(params.query("_internal_free")[["lower", "upper"]].to_numpy().T)
+    bounds = tuple(
```

Comment:
Maybe a function for this thing because it is done two times and if we had not forget that it exists in two places we might have fixed only one call?

Reply:
That makes sense to me since I think it contains exactly the same logic across the two examples!

Reply:
Done. Thanks!